docs/plugins/storage.md

```diff
@@ -31,12 +31,57 @@ initial state can be picked up by those plugins.
 ### Options
 The plugin has the following optional values:
 
-- `key`: Default key to persist. You can pass a string or array of strings that can be deeply nested via dot notation. If not provided, it defaults to all states using the `@@STATE` key.
+- `key`: State name to be persisted. You can pass a string or array of strings that can be deeply nested via dot notation. If not provided, it defaults to all states using the `@@STATE` key.
 - `storage`: Storage strategy to use. This defaults to LocalStorage but you can pass SessionStorage or anything that implements the StorageEngine API.
 - `deserialize`: Custom deserializer. Defaults to `JSON.parse`
 - `serialize`: Custom serializer, defaults to `JSON.stringify`
 - `migrations`: Migration strategies
 
+### Key option
+
+The `key` option is used to determine what states should be persisted in the storage. `key` shouldn't be a random string
+or array of string, it has to coincide with your state names. Let's look at the below example:
```

Comment:
Don't put your own line breaks in. The markdown will handle this when rendered.
Also removed the mention about arrays because it is confusing to say that it is not valid, where it actually is.
```suggestion
, it has to coincide with your state names. Let's look at the below example:
```